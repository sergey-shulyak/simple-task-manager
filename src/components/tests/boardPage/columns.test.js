import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { tickets } from '../testData';

import Column from '../../boardPage/board/column';

const props = {
  title: 'Sample title',
  tickets,
  className: 'custom',
  boardId: '42'
};

describe('Column', () => {
  it('Should render correctly', () => {
    const wrapper = shallow(<Column {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
