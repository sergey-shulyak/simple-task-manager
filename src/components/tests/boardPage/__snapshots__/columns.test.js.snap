// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column Should render correctly 1`] = `
<div
  className="column custom"
>
  <h2
    className="column__title"
  >
    Sample title
  </h2>
  <Ticket
    boardId="42"
    className="column__ticket"
    columnId="1"
    description=""
    id="1"
    key="1"
    priority="medium"
    title="Test ticket 1"
    type="task"
  />
  <Ticket
    boardId="42"
    className="column__ticket"
    columnId="2"
    description=""
    id="2"
    key="2"
    priority="medium"
    title="Test ticket 1"
    type="task"
  />
  <Ticket
    boardId="42"
    className="column__ticket"
    columnId="3"
    description=""
    id="3"
    key="3"
    priority="medium"
    title="Test ticket 1"
    type="task"
  />
</div>
`;
